iosxr_test.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/python
+'''
+Author: Rich Wellum (richwellum@gmail.com)
+
+A tool to run some basic tests on a IOS XRv vagrant box, using pxssh module.
+
+This will verify both IOS XR Linux and IOS XR Console access.
+
+Is called from iosxr_iso2vbox post generation to verify sanity of
+the Virtualbox,
+
+Can also be run manually, like this:
+
+python iosxr_test.py iosxrv-fullk9-x64.box
+'''
+
+from __future__ import print_function
+import sys
+import pexpect
+from pexpect import pxssh
+import subprocess
+import argparse
+import os
+from iosxr_iso2vbox import run
+
```

Comment:
In the long run it would be nice to refactor this to use Python's `unittest` module, as it will make for easier integration with things like Travis/Jenkins.